examples/ini.js

```diff
@@ -0,0 +1,47 @@
+const fs = require("fs");
+const ini = require("ini");
```

Comment:
Please use single quotes:

```suggestion
const ini = require('ini');
```